frontend/GoodGrocer/app/screens/RequestDetail.js

```diff
@@ -1,73 +1,75 @@
-import React, {useEffect, useState} from 'react';
-import { SafeAreaView, StyleSheet, Text, Image, ScrollView, View } from 'react-native';
-import ItemCard from '../components/ItemCard'
-import useRequest from '../hooks/useRequest';
+import React, { useEffect, useState } from "react";
+import {
+  SafeAreaView,
+  StyleSheet,
+  Text,
+  Image,
+  ScrollView,
+  View,
+} from "react-native";
+import ItemCard from "../components/ItemCard";
+import useRequest from "../hooks/useRequest";
 
 function RequestDetail(props) {
-  const [items, setItems] = useState([]); 
+  const items = props.route.params.items;
 
-  const getItemsInRequest = useRequest({
-    url: `/request/items/${props.route.params.requestId}`,
-    method: "get",
-    onSuccess: (data) => {
-      console.log(data)
-     data.forEach((item)=>{setItems(oldArray=>[...oldArray, {itemName: item.name, numOfItem: item.quantity+item.quantity_type, id: item.id}])}); 
-    }
-  });
-  const func = async () => getItemsInRequest.doRequest(); 
-  useEffect(()=> {func()},[]); 
-
-
-  const itemList = items.map((item) => <View style={styles.itemCard} key={item.id}><ItemCard itemName={item.itemName} numOfItem={item.numOfItem} /></View>);
-    return <SafeAreaView style={styles.container}>
-    <View style={styles.content}>
-    <Image source={require("../assets/logo.png")}/>
-    <Image style={styles.profileImage} source={{uri: "https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460__340.png"}}/>
-    <Text>Store: {props.route.params.storeName}</Text>
-    <Text>Item Count: {items.length}</Text>
-    <ScrollView style={styles.listOfItems}>
-        {itemList}
-      </ScrollView>
+  const itemList = items.map((item) => (
+    <View style={styles.itemCard} key={item.id}>
+      <ItemCard itemName={item.itemName} numOfItem={item.numOfItem} />
     </View>
-  </SafeAreaView>; 
-
+  ));
+  return (
+    <SafeAreaView style={styles.container}>
+      <View style={styles.content}>
+        <Image source={require("../assets/logo.png")} />
+        <Image
+          style={styles.profileImage}
+          source={{
+            uri: "https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460__340.png",
```

Comment:
Should we change this to pass in the user's actual profile pic?

Reply:
Will do it another PR.